docs/diagrams/AddMeetingCommandSequence.puml

```diff
@@ -0,0 +1,45 @@
+@startuml
```

Comment:
Diagram looks very small on the site itself. Although zooming in does help with deciphering what is written in the diagram. Could be better by using SD in sequence diagrams so as to shift so parts of the diagram to another picture to make it more visible without the need to zoom.

Reply:
![image](https://user-images.githubusercontent.com/77721568/160959273-513911f4-05fe-41ba-9e52-b4087bb59293.png)